$NetBSD: patch-ab,v 1.9 2004/04/21 13:23:57 markd Exp $

Use of tainted arguments in exec is deprecated at /var/www/cgi-bin/cvsweb.cgi line 2043.
===================================================================
RCS file: /home2/webcvs/mirror/kdelibs/kdesu/su.cpp,v
retrieving revision 1.32.2.2
retrieving revision 1.32.2.3
diff -u -p -r1.32.2.2 -r1.32.2.3
--- kdesu/su.cpp	2004/03/26 23:26:45	1.32.2.2
+++ kdesu/su.cpp	2004/04/13 13:32:27	1.32.2.3
@@ -78,9 +78,9 @@ int SuProcess::exec(const char *password
 	args += "root";
     else
 	args += m_User;
-    args += "-";
     args += "-c";
     args += QCString(__KDE_BINDIR) + "/kdesu_stub";
+    args += "-";
 
     QCString command = __PATH_SU;
     if (::access(__PATH_SU, X_OK) != 0)
